src/main/java/coresearch/cvurl/io/multipart/Part.java

```diff
@@ -17,8 +17,10 @@
  * Represent part of multipart data.
  */
 public class Part<T extends Part<T>> {
-    public static final String CRLF = "\r\n";
-    public static final String BOUNDARY_DELIMITER = "--";
+
+    private static final String CRLF = "\r\n";
+    private static final String BOUNDARY_DELIMITER = "--";
+    private static final String CONTENT_PART = "content";
```

Comment:
"content" is used in null checks as argument name so if you want to extract it probably should be named CONTENT or CONTENT_ARGUMENT or smth like this.

Reply:
Agreed.